Adapter_Revit_UI/CRUD/Create.cs

```diff
@@ -42,225 +40,55 @@ public partial class RevitUIAdapter
 
         protected override bool ICreate<T>(IEnumerable<T> objects, ActionConfig actionConfig = null)
         {
-            if (Document == null)
-            {
-                NullDocumentCreateError();
-                return false;
-            }
-
-            if (objects == null)
-            {
-                NullObjectsCreateError();
-                return false;
-            }
-
             if (objects.Count() == 0)
                 return false;
-            
-            RevitPushConfig pushConfig = actionConfig as RevitPushConfig;
-            if (pushConfig == null)
-            {
-                BH.Engine.Reflection.Compute.RecordWarning("Revit Push Config has not been specified. Default Revit Push Config is used.");
-                pushConfig = RevitPushConfig.Default;
-            }
-
-            bool suppressFailureMessages = pushConfig.SuppressFailureMessages;
 
+            IEnumerable<IBHoMObject> bhomObjects = objects.Cast<IBHoMObject>();
```

Comment:
This should not be needed, since you are overriding the base ICreate method, that has this signature:
```cs
 ICreate<T>(IEnumerable<T> objects, ActionConfig actionConfig = null) where T : IBHoMObject
```
so `T` can only be `BHoMObject`. 

Correct me if wrong

Reply:
@alelom I tried to change it but had some weird issues with casting - will have another look

Reply:
Oke, double checked: this leads to an interesting case of the compiler not being able to cast from `IEnumerable<IBHoMObject>` to `IEnumerable<BH.oM.Base.IBHoMObject>`, which I do not understand tbh. @IsakNaslundBh FYI

![image](https://user-images.githubusercontent.com/26874773/75795565-ca4e3780-5d72-11ea-9ece-731d69296aff.png)

Reply:
It's generics stuff. You can leave this.